iati/tests/base_functional_tests.py

```diff
@@ -1 +1,20 @@
 """A module of functional tests for base site functionality."""
+import pytest
+from conftest import LOCALHOST
+
+
+DEFAULT_PAGE_NAMES = [
+    'about',
+    'contact',
+    'events',
+    'news',
+    'guidance_and_support'
```

Comment:
Yep, I can see where keeping the naming consistent would come in handy. In the short-term, if we don't want to bother Fffunction with making a name change or doing it ourselves, we could make this a keyed dictionary like the createdefaultpages command. Or we could even import the page names from the createdefaultpages command.

Reply:
I guess there are a couple of things to consider here:
- We need to deal with naming consistency anyway so at some point we will have to bother fffunction if we can't make the changes ourselves. How do we work this into our work flow?
- Extracting that list from `createdefaultpages` will require refactoring the code before we've unit tested it. While extracting it would be a good idea, is it something we should wait to do when we have better test coverage?